packages/devtools_app/lib/src/service_extensions.dart

```diff
@@ -51,7 +52,8 @@ class ToggleableServiceExtensionDescription<T>
 
 class ServiceExtensionDescription<T> {
   ServiceExtensionDescription({
-    this.icon,
+    this.disabledIcon,
+    this.enabledIcon,
```

Comment:
it seems there are a few cases where enabledIcon and disabledIcon are the same. Perhaps we should have `disabledIcon` default to `enabledIcon` when `enabledIcon != null` and disabled icon is not specified.

Reply:
Yep, I will make `enabledIcon` required and default `disabledIcon` to it if not specified